test_runner/fixtures/zenith_fixtures.py

```diff
@@ -64,15 +64,15 @@ def pytest_configure(config):
          raise Exception('Too many workers configured. Cannot distrubute ports for services.')
 
     # does not use -c as it is not supported on macOS
-    cmd = ['pgrep', 'pageserver|postgres|wal_acceptor']
+    cmd = ['pgrep', 'pageserver|postgres|safekeeper']
```

Comment:
Although there may be some old versions called `wal_acceptor` running locally or in prod, I assume they will be dropped in full very quickly and should not cause any disturbance after few weeks.